app.js

```diff
@@ -22,7 +26,7 @@ app.use((err, req, res, next) => {
 })
 
 app.all("/*", (req, res) => {
```

Comment:
I'd suggest keeping your endpoint functions together and having your error handling functions come afterwards